go/install/install_darwin.go

```diff
@@ -17,6 +15,7 @@ import (
 
 	"github.com/blang/semver"
 	"github.com/kardianos/osext"
+	kbnmInstaller "github.com/keybase/client/go/kbnm/installer"
```

Comment:
Yea I agree about package names, but I felt like a local alias would have more local variable naming rules rather than exported package naming rules. Meeh I'll change it.